components/SignIn/Profile/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Image from "next/image";
+import {useSession, signIn, signOut} from "next-auth/react";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import styled from "styled-components";
+import Button from "@mui/material/Button";
+
+const ProfileImage = styled.img`
+	border-radius: 50%;
+	width: 36px;
+	height: 36px;
+`;
+
+export default function Profile() {
+	const {data: session, status} = useSession();
+
+	return (
+		<>
+			{session ? (
+				<Button
+					variant="text"
+					color="inherit"
+					onClick={() => signOut()}>
+					<ProfileImage src={session?.user?.image} alt="#" />
+					<span>Sign out</span>
```

Comment:
Same here with the `span`: [the docs of mui ](https://mui.com/material-ui/react-button/#basic-button)don't use a `span` for a text button. But please correct me if I'm wrong, I've not worked with mui before.